### DISCUSSION TITLE
Getting strange error when trying to delete or save a row.

### DISCUSSION THREAD

**Discussion by adamsanclemente** [Budibase/budibase#4204] [Help]
Whenever I try to save or delete a row I get this weird error code.

`AccessDeniedAccess Denied.289142537011068929/28914253701106892916CDA5FC3C48532F7694be21-ff79-4617-a2d4-3a8f1f74e79a`
![sadsdasdasd](https://user-images.githubusercontent.com/26911654/151075352-74754527-4119-416a-aed3-fbcb896e86a7.gif)

**Comment by mjashanks** [comment]
Check your access on the table that you are Updating/Deleting - 

![image](https://user-images.githubusercontent.com/3524181/151075662-6c426969-f353-4a4a-a97f-959692db3194.png)

Of course, if you are developing an application, you are probably a full admin, so this shouldn't matter. It depends on who you are logged in with

**Reply by adamsanclemente** [reply]
I figured that that was the issue, so I set it to public, still does the same error.

**Reply by mjashanks** [reply]
Actually, it looks like the `Delete` button is trying to redirect you?

Check the button actions. You probably just want to `Close Screen Modal` after deletion.

**Reply by adamsanclemente** [reply]
That got rid of the strange issue there. But I still am getting a red error whenever I try and delete something from the button.

edit: 

![image](https://user-images.githubusercontent.com/26911654/151077527-7836e7b2-d416-4b71-bcd8-05539926b887.png)

**Reply by mjashanks** [reply]
Just getting back to this - are you still having an issue?

If you 
1. open up your browser's developer tools
2. goto the network tab
3. push the delete button
4. look for a failed request (in red)
5. Click on it
6. Paste the response in here - hopefully there is a more detailed error message!
